integration/src/main/resources/configs/perf/DataReferenceLifecycle.json

```diff
@@ -14,5 +14,5 @@
       "parameters": ["wm-default-spend-profile", "97b5559a-2f8f-4df3-89ae-5a249173ee0c", "terra"]
     }
   ],
-  "testUserFiles": ["william.json"]
+  "testUserFiles": ["bella.json"]
```

Comment:
How do you decide when to use Liam vs Bella?

Reply:
Bella has access to the dev TDR snapshot we use in tests, so I used her for tests that talk to TDR. Users are pretty interchangeable otherwise, so I picked Liam arbitrarily for the other ones